README.md

```diff
@@ -49,6 +56,21 @@ make
 make test
 ```
 
+Optionally, you can install [DeepMind Lab](https://deepmind.com/blog/open-sourcing-deepmind-lab) and [Project Malmo](https://www.microsoft.com/en-us/research/project/project-malmo).
```

Comment:
Before talking about DeepMind Lab and Malmo, say something like "By default, XWorld only builds the first four games: SimpleGame, SimpleRace, Atari, and XWorld2D."